### ISSUE TITLE
missing file

### ISSUE THREAD

**Issue by sashank06** [YanWenqiang/HBLSTM-CRF#1]
Hi
Do you have the swda_data file? That is missing from the repository

**Comment by YanWenqiang** [comment]
I am sorry data preprocessing file swda_data.py was removed by mistake. You can preprocess the original data and convert dialogues to the form like line #L250 and labels to the form like line #L251.

**Comment by hughperkins** [comment]
Dont suppose ... could we add swda_data.py file to the repo please?

**Comment by shzamani** [comment]
Hi, can you please add the swda_data.py file to the repo? Thanks.

**Comment by adamelen** [comment]
> Hi, can you please add the swda_data.py file to the repo? Thanks.

Yes please, it would be so helpful. There are many questions about how exactly the data is used.
Thanks in advance.
